src/AppKit/AppKitThreadAccessException.cs

```diff
@@ -1,7 +1,11 @@
 using System;
+using System.Runtime.Versioning;
 
 namespace AppKit
 {
+#if NET
+	[SupportedOSPlatform ("macos")]
+#endif
```

Comment:
I would think `AppKitThreadAccessException` would be available on macOS as well, since AppKit is available on macOS.

Reply:
Fixed.